src/screens/Game/index.tsx

```diff
@@ -1,6 +1,8 @@
 // Packages
 import { useEffect, useState } from 'react';
 import { useColorScheme, StatusBar, Text, View } from 'react-native';
+import { MW_API_URL, MW_API_KEY } from '@env';
```

Comment:
I think this should be in the constants import section. I can share my import struct snippet with you.